redbot/cogs/streams/streams.py

```diff
@@ -283,37 +284,53 @@ async def streamalert(self, ctx: commands.Context):
         pass
 
     @streamalert.group(name="twitch", invoke_without_command=True)
-    async def _twitch(self, ctx: commands.Context, channel_name: str = None):
+    async def _twitch(
+        self,
+        ctx: commands.Context,
+        channel_name: str = None,
+        discord_channel: discord.TextChannel = None,
+    ):
         """Manage Twitch stream notifications."""
         if channel_name is not None:
-            await ctx.invoke(self.twitch_alert_channel, channel_name)
+            await ctx.invoke(self.twitch_alert_channel, channel_name, discord_channel)
         else:
             await ctx.send_help()
 
     @_twitch.command(name="channel")
-    async def twitch_alert_channel(self, ctx: commands.Context, channel_name: str):
+    async def twitch_alert_channel(
+        self, ctx: commands.Context, channel_name: str, discord_channel: discord.TextChannel
```

Comment:
`discord_channel` should have a default of `None` here same as other commands.
```suggestion
        self, ctx: commands.Context, channel_name: str, discord_channel: discord.TextChannel = None
```